code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -5,6 +5,7 @@
 /datum/reagent/medicine/on_mob_life(mob/living/M)
 	current_cycle++
 	holder.remove_reagent(id, (metabolization_rate / M.metabolism_efficiency) * M.digestion_ratio) //medicine reagents stay longer if you have a better metabolism
+	return STATUS_UPDATE_NONE
```

Comment:
@Crazylemon64 I presume this is what we want to do here; the medicine reagents completely override `/datum/reagent/on_mob_life`, so I'm guessing they technically need this, yes?

Reply:
@Fox-McCloud This is correct. Although since it's a bitfield, the default value should be 0 anyways, but this line should be useful for declaring intent (`STATUS_UPDATE_NONE` is 0)